### DISCUSSION TITLE
Joystick buttons - remapping button 11 and 12

### DISCUSSION THREAD

**Discussion by aykaramba** [nesbox/TIC-80#1439] [General]
Hi.

I am learning TIC80 and have been using a Logitech F710 Gampad with various games.  One of the things that I noticed is that the analog sticks have button 11 and button 12 under the pads and users keep on pressing them all the time and getting kicked out to menu.

Why is TIC80 reacting to button 11 and 12 in any way?  

How is this addressed?  I have tried using qjoypad to remap the buttons but TIC80 still drop the user to menu.

Are we as game devs supposed to disable buttons 11 and 12 programmatically or is this handled at the level of TIC80 or the operating system? 

I would love to have users be able to grab a joystick and just button smash and have some fun, so just trying to learn where in the stack this is handled.

**Comment by nesbox** [comment]
Hello,

it doesn't work inside TIC correctly, MENU will appear when any unhandled button is pressed
of course, we have to fix this, I think one of the obvious solutions is to add mapping of the MENU button to the config (as part of #1154).
We will try to fix it in the next 0.90 release.
Thank you

**Reply by aykaramba** [reply]
Oh!  Okay, well that makes sense!  I greatly appreciate the feedback, this was driving me batty.

Thank you kindly for the great work.
